_episodes/01-intro.md

```diff
@@ -167,66 +191,74 @@ stuck on it](../fig/python-sticky-note-variables-02.svg)
 >
 > Let's now change `weight_kg`:
 >
-> ~~~
+> ```
 > weight_kg = 100.0
 > print('weight in kilograms is now:', weight_kg, 'and weight in pounds is still:', weight_lb)
-> ~~~
+> ```
+>
 > {: .language-python}
 >
-> ~~~
+> ```
 > weight in kilograms is now: 100.0 and weight in pounds is still: 143.0
-> ~~~
+> ```
+>
 > {: .output}
 >
 > ![Value of 100.0 with label weight_kg stuck on it, and value of 143.0 with label weight_lb
 stuck on it](../fig/python-sticky-note-variables-03.svg)
 >
 > Since `weight_lb` doesn't "remember" where its value comes from,
 > it is not updated when we change `weight_kg`.
-{: .callout}
-
+> {: .callout}
 
 > ## Check Your Understanding
 >
 > What values do the variables `mass` and `age` have after each of the following statements?
 > Test your answer by executing the lines.
 >
-> ~~~
+> ```
 > mass = 47.5
 > age = 122
 > mass = mass * 2.0
 > age = age - 20
-> ~~~
+> ```
+>
 > {: .language-python}
 >
 > > ## Solution
-> > ~~~
+> >
+> > ```
 > > `mass` holds a value of 47.5, `age` does not exist
 > > `mass` still holds a value of 47.5, `age` holds a value of 122
 > > `mass` now has a value of 95.0, `age`'s value is still 122
 > > `mass` still has a value of 95.0, `age` now holds 102
-> > ~~~
+> > ```
+> >
 > > {: .output}
-> {: .solution}
-{: .challenge}
+> > {: .solution}
+> > {: .challenge}
 
 > ## Sorting Out References
 >
-> What does the following program print out?
+> Python allows you to assign multiple values to multiple variables in one line by separating
+> the variables and values with commas. What does the following program print out?
```

Comment:
I think this text could be helpful for users, since we did not cover this in the lesson content. It does not get into technical detail, but at this point in the lesson, I think that makes sense.